plugins/pom.xml

```diff
@@ -80,6 +82,32 @@
       <scope>provided</scope>
       <optional>true</optional><!-- no need to have this at runtime -->
     </dependency>
+    <!-- mark logging dependencies as provided by war -->
+    <dependency>
+      <groupId>org.slf4j</groupId>
+      <artifactId>slf4j-api</artifactId>
+      <version>${slf4jVersion}</version>
+      <scope>provided</scope>
```

Comment:
I thought it would be a good idea to include this as a dependency, so people know, that they should use slf4j for their own logging purposes. As it is marked as provided, it will not be included in the HPI but is available during compile and test.